### ISSUE TITLE
InputSelect loses class attribute when setting disabled equals true

### ISSUE THREAD

**Issue by jan-johansson-mr** [aspnet/AspNetCore#16570]
### Describe the bug
InputSelect component loses it's class attribute when setting disabled attribute to true, e.g.

`<InputSelect disabled="@(isRemoved)">...</InputSelect>`

### To Reproduce
1. Using this version of ASP.NET Core 3.1 preview and Visual Studio preview

### Expected behavior
The component should not drop class attribute for the select control. I've not seen other input components having the same behavior.

### Additional context
Include the output of `dotnet --info`
  3.1.100-preview1-014459 [C:\Program Files\dotnet\sdk]
  Microsoft.NETCore.App 3.1.0-preview1.19506.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.0-preview1.19506.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

**Comment by javiercn** [comment]
@jan-johansson-mr thanks for contacting us.

@SteveSandersonMS Do you know why this can be?

**Comment by SteveSandersonMS** [comment]
No, I don't know. It would be great to have a complete repro. The code posted above doesn't have any `class` attribute.

**Comment by jan-johansson-mr** [comment]
Hi,

I've just done an example, derived from the blazor server side template, that I can upload (solution with the source files), to show InputSelect dropping class when disabled.

Please have a look at the [example](https://github.com/jan-johansson-mr/BlazorExample).

Thanks

**Comment by SteveSandersonMS** [comment]
Well this is very surprising. We almost never get bugs in the actual rendering system, but this really is one. It requires a very particular combination of things to trigger it. Will be fixed in https://github.com/aspnet/AspNetCore/pull/16640

**Comment by jan-johansson-mr** [comment]
Thanks!!

**Comment by SteveSandersonMS** [comment]
Reopening until PR is merged.
